pyhunter/pyhunter.py

```diff
@@ -10,20 +10,27 @@ def __init__(self, api_key):
         self.base_endpoint = 'https://api.hunter.io/v2/{}'
 
     def _query_hunter(self, endpoint, params, request_type='get',
-                      payload=None):
-        if not payload:
-            res = getattr(requests, request_type)(endpoint, params=params)
-        else:
-            res = getattr(requests, request_type)(endpoint, params=params,
-                                                  json=payload)
+                      payload=None, headers=None, raw=False):
+
+        request_kwargs = dict(params=params)
+        if payload:
+            request_kwargs.setdefault(json=payload)
+
+        if headers:
+            request_kwargs.setdefault(headers=headers)
+
+        res = getattr(requests, request_type)(endpoint, **request_kwargs)
```

Comment:
This introduced the issue https://github.com/VonStruddle/PyHunter/issues/53
dict.setdefault() takes no keyword arguments, the correct syntax is `setdefault(key[, default])` according to [the docs](https://docs.python.org/3/library/stdtypes.html#dict.setdefault)